lms/static/scripts/frontend_apps/components/dashboard/test/DashboardActivityFilters-test.js

```diff
@@ -252,6 +331,21 @@ describe('DashboardActivityFilters', () => {
       });
     });
 
+    it('shows 1 selected item when a single unknown one is selected', () => {
+      const wrapper = createComponent({
+        selectedCourseIds: ['999'],
+        selectedAssignmentIds: ['999'],
+        selectedStudentIds: ['999'],
+      });
+
+      assert.equal(getSelectContent(wrapper, 'courses-select'), '1 course');
+      assert.equal(
+        getSelectContent(wrapper, 'assignments-select'),
+        '1 assignment',
+      );
+      assert.equal(getSelectContent(wrapper, 'students-select'), '1 student');
+    });
```

Comment:
This seems like it is separate from the other changes in this PR?

Reply:
Perhaps in the way the test is written, yes. When the first page does not contain some of the selected IDs, we can't display their human-friendly values, so the fallback is "1 student/course/assignment".

However, that would also incidentally happen for invalid IDs, which is something that I plan to addressed separately.

Perhaps I should write the test in a way that clearly references the first case, and not the second. Let me give it a thought.